src/css/components/_appstore-menu.scss

```
@@ -1,15 +1,12 @@
 .appstore-menu {
     width: 100%;
     height: $master-height - 75px;
-    overflow-y: scroll;
+    overflow-y: hidden;
```

```suggestion
    overflow-y: scroll;
```

It should still be possible to scroll here, right?